packages/terra-embedded-content-consumer/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Added
   * Added screenreader support to announce context of embedded iframe content.
+  * Added visual focus indicator on the iframe when the content doesn't have any inteactable element, iframe is scrollable, and content is scrollable for keyboard only users
```

Comment:
nit: needs a period at the end

Reply:
Fixed with https://github.com/cerner/terra-framework/pull/1874/commits/10ee5fc7493b71f8c90e1e097bc16610a5f9d3b2